### ISSUE TITLE
Ligature "=>" isn't displayed correctly

### ISSUE THREAD

**Issue by pacmancoder** [zargony/atom-language-rust#62]
I found interesting font [Fira Code](https://github.com/tonsky/FiraCode) with ligatures, but I found that ligature for operator "=>" is not displayed correctly. So, I tried to fix this. I was getting correct ligature if I changed 'match' for operators section form this:
```cson
# Operators
{
  'comment': 'Operator'
  'name': 'keyword.operator.rust'
  'match': '(\\+|-|/|\\*|=|<=|>=|\\^|&|\\||!|%|::|\\bas\\b|<<|>>|>|<)'
}
```
to this *(added |=>| )*:
```cson
# Operators (Good)
{
  'comment': 'Operator'
  'name': 'keyword.operator.rust'
  'match': '(\\+|-|/|\\*|=>|=|<=|>=|\\^|&|\\||!|%|::|\\bas\\b|<<|>>|>|<)'
}

```  

 Correct | Incorrect
 ----------- | ---------  
![Correct](https://cloud.githubusercontent.com/assets/3994505/11975678/fa29ab9a-a978-11e5-9470-5f9309cc04d0.png) | ![Incorrect](https://cloud.githubusercontent.com/assets/3994505/11975683/082042ea-a979-11e5-9882-1473424fc791.png)

So, is my fix correct?  
(Sorry for my poor English)

**Comment by Victorystick** [comment]
It should be correct. It is however a duplicate of #59.

**Comment by pacmancoder** [comment]
This should be fixed in #59
